tests/test.cpp

```diff
@@ -1228,6 +1255,7 @@ int main(int /*argc*/, const char * /*argv*/[]) {
   UnknownFieldsTest();
   ParseUnionTest();
   ConformTest();
+  FixedLengthArrayTest();
 
   if (!testing_fails) {
     TEST_OUTPUT_LINE("ALL TESTS PASSED");
```

Comment:
we're missing documentation. use of arrays should be added to the schema docs, and the tutorial.